Jenkinsfile

```diff
@@ -42,8 +42,11 @@ pipeline {
       }
     }
     stage('Publish to CDN') {
-      when {
-        branch 'master'
+      when { 
+        anyOf { 
+          branch 'beta'
```

Comment:
Will we revert this eventually or keep for future use?

Reply:
Either way, probably can keep it.